layer/USAGE_android.md

```diff
@@ -1,143 +1,151 @@
-Errors To Stderr | debug.gfxrecon.log_errors_to_stderr | BOOL | This option allows you to force all error messages that would be normally logged to also output to stderr.
-Log File | debug.gfxrecon.log_file | STRING | This option allows you to define the path and name of a log file that will be generated with log messages.
-Log File Create New | debug.gfxrecon.log_file_create_new | BOOL | This option indicates that you want to either create a new file every time the layer is triggered, or append to the old log file.
-Log File Flush After Write | debug.gfxrecon.log_file_flush_after_write | BOOL | This option allows you to force a flush after every log file write to make sure you don't loose messages in a buffer.
-Log File Keep Open | debug.gfxrecon.log_file_keep_open | BOOL | This option forces the log file to remain open after it's created to allow for faster recording of log messages.
-Log Level | debug.gfxrecon.log_level | STRING | This option allows you to choose what log level you desire to trigger.  Available options include: "debug", "info", "warning", "error", and "fatal".  Any level selected will include all levels listed after it.  For example, choosing "warning" will also log out "error" and "fatal" messages.
-Log Output To Console | debug.gfxrecon.log_output_to_console | BOOL | This option allows log messages to be written out to stdout (or whatever debug console is available on the target platform.
-Memory Tracking Mode | debug.gfxrecon.memory_tracking_mode | STRING | This option allows the user to determine what memory tracking mode the layer uses when handling memory.  Available options are: "page_guard", "assisted" and "unassisted".  <ul><li>"page_guard" is used to determine which regions of memory to write on an `vkUnmapMemory` or `vkQueueSubmit` call.  "page_guard" also shadows uncached memory so as to properly provide all memory it can.</li><li>"assisted" assumes the application will always flush after writing to mapped memory, so will only write on a flush.</li><li>"unassisted" assumes the application does not flush, so writes all mapped data on an `vkUnmapMemory` or `vkQueueSubmit` call.</li></ul>
+Capture File Name | debug.gfxrecon.capture_file | STRING | Path to use when creating the capture file.  Default is: "/sdcard/gfxrecon_capture.gfxr".
+Capture File Compression Type | debug.gfxrecon.capture_compression_type | STRING | Compression format to use with the capture file.  Valid values are: "LZ4", "ZLIB", and "NONE". Default is: "LZ4".
+Capture File Timestamp | debug.gfxrecon.capture_file_timestamp | BOOL | Add a timestamp to the capture file with the form: "gfxrecon_capture_yyyymmddThhmmss.gfxr".  Default is: true.
+Log Allow Indents | debug.gfxrecon.log_allow_indents | BOOL | Apply additional indentation formatting to log messages.
+Log Break On Error | debug.gfxrecon.log_break_on_error | BOOL | Trigger a debug break when logging an error.
```

Comment:
The default values here and below seem "obvious"... but I'd prefer they be explicitly indicated.